test/studies/graph500/kristyn/Graph500_1D_onV/Graph500_Modules/Verify.chpl

```diff
@@ -139,19 +139,19 @@ proc compute_levels (ParentTree, Level): int
        if (error >= 0 && parent < 0) then error = -2;
 
        // Assign levels until we meet an already-leveled vertex
-   
+
        if ( error == 0 ) {
          nhop += Level[parent];
          parent = k;
 
          while ( Level[parent] < 0) {
            Level[parent] = nhop;
-           nhop -= 1; 
+           nhop -= 1;
            parent = ParentTree[parent];
          }
        }
 
-     } 
+     }
```

Comment:
Whitespace ditto in test/studies/graph500

Would be much nicer in a separate PR.